### DISCUSSION TITLE
Stack Component in Docs

### DISCUSSION THREAD

**Discussion by tannerhallman** [react-native-elements/react-native-elements#3627] [General]
Hi there, Is the `Stack` component released yet? I don't see it documented and I would like to bring up think it's quite confusing to include it in your docs examples if that's not a valid export yet.

I am using Version: 4.0.0-rc.6.

Example:
https://reactnativeelements.com/docs/components/button#variants

**Comment by arpitBhalla** [comment]
you can find it on [npm](https://www.npmjs.com/package/@rneui/layout), would be available on website, once released as beta/stable

**Comment by soullivaneuh** [comment]
@arpitBhalla Thanks for the hint! However, it was hard to find out this discussion, looking also for this `Stack` component.

May I suggest you to add a tiny section on the main documentation referencing this package? :+1:

**Reply by arpitBhalla** [reply]
Working on that

**Comment by tannerhallman** [comment]
@arpitBhalla Shall I convert this to an issue for tracking?

**Reply by tannerhallman** [reply]
https://github.com/react-native-elements/react-native-elements/issues/3771
